### DISCUSSION TITLE
Chris Graber in US

### DISCUSSION THREAD

**Discussion by christophergraber** [Green-Software-Foundation/discussions#171] [Introductions]
I'm Chris Graber (he/him) and have a varied background in technology and nonprofit spaces.

Here's my [LinkedIn](https://www.linkedin.com/in/chris-graber-5b33a32/) and I also run [DIY Eco Guy](https://diyecoguy.com) which is where I share DIY home sustainability projects I've worked on like grey water systems, electrification, etc.

My interest in Green Software is deep, coming from a foundation of loving nature. I've always wanted to find ways for us humans to impact the world in a positive way! If we can make software more sustainable that is a great thing. Thanks to all who are involved in this.

**Comment by NAMRATA-WOKE** [comment]
@christophergraber nice to have you here! **[Do you receive our weekly newsletter](https://grnsft.org/newsletter)**? It's filled with good stuff.
